### PR TITLE
fix: fixed unmute btn function call on samples

### PR DESCRIPTION
**Pull request by meta-dipanshu-sharma**
# COMPLETES #<[ Spark Issue](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-311027) >

## This pull request addresses

Unmute button fix in samples app.

## by making the following changes

Updated function call to unmuteAudio() on unmute button on [samples app](https://webex.github.io/webex-js-sdk/samples/browser-plugin-meetings).

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import sinon from 'sinon';
 import {assert} from '@webex/test-helper-chai';
 import MockWebex from '@webex/test-helper-mock-webex';
+
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js:4]: unnecessary whitespace change, it shouldn't be part of this PR

**Review by marcin-bazyl — APPROVED** [review]

**Comment by marcin-bazyl** [comment]
Just wondering if there are any samples app tests that should be updated to check muting/unmuting.

**Review by mkesavan13 — APPROVED** [review]

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js (hunk 1 above)
Comment (meta-dipanshu-sharma) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js:4]: @marcin-bazyl During rebasing I got this extra change and I was not able to remove it due to eslint rules specified.

**Comment by meta-dipanshu-sharma** [comment]
> Just wondering if there are any samples app tests that should be updated to check muting/unmuting.

I didn't find any sample test related to muting/unmuting audio.

**Review by arun3528 — APPROVED** [review]
i would do a reset on the file and it should fix the white space
